File: f77/build08/diffeq.f
C
      Subroutine Differential_Equations
C
C      This routine calls the modules that evaluate the state derivatives for the
C      terminal homing simulation
C
      Call Airframe_Response
      Call Target
      Call Seeker
      Call Flight_Computer
      Call Kinematics
C
      Return
      End

File: gcc/build08/diffeq.c
#include "diffeq.h"
#include "airframe.h"
#include "fcomp.h"
#include "kmtc.h"
#include "seeker.h"
#include "target.h"
void differential_equations() {
  // This routine calls the modules that evaluate the state derivatives for the
  // terminal homing simulation.
  airframe_response();
  target();
  seeker();
  flight_computer();
  kinematics();
}
